.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-*.config.ts
\ No newline at end of file
+*.config.ts
+apps/postybirb-ui/src/icons/icons.ts
```

Comment:
Configs are ignored because they too often use externeous imports (maybe i should disable that in eslintrc, huh)

Icons are ignored because import/organize imports and few other rules (only now i realize that this can be disabled fir whole file using comments). I think my next pr will include this fixes.

Reply:
Fixed